### DISCUSSION TITLE
Radix select shifts the whole layout to the right in chrome

### DISCUSSION THREAD

**Discussion by Biki-das** [radix-ui/primitives#2101] [Help]
## Bug report
While using the Radix select component, the select component is shifting the layout to the right, this happens only in chromium based browsers (edge and chrome), the same does not occur in firefox.

### Current Behavior

https://user-images.githubusercontent.com/72331432/232389549-b87a7eeb-168c-4896-9f34-bd14fbd9f006.mp4

### Expected behavior

https://user-images.githubusercontent.com/72331432/232389798-1ed0fedd-2bc6-49ad-a314-71310f67f199.mp4

### Reproducible example

[CodeSandbox Template](https://codesandbox.io/s/2r30e)

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |      select   |         |
| React            | 18     |         |
| Browser          |    chrome     |         |
| Assistive tech   |         |         |
| Node             | 17   |         |
| npm/yarn         |         |         |
| Operating System |    windows     |         |

**Comment by aokigit** [comment]
From what I can see in the video, this happens cause the scrollbar disappears once the select opens.

**Comment by Biki-das** [comment]
> From what I can see in the video, this happens cause the scrollbar disappears once the select opens.

What could be the reason?

**Comment by andy-hook** [comment]
> What could be the reason?

@Biki-das  Radix will set a margin on `body` equal to the scrollbar width when select opens, I would start by checking that this rule is being applied correctly in the browser.

If you can link your project we might be able to help further.

**Reply by Biki-das** [reply]
@andy-hook  have not deployed the app but can share a codesandbox example, will it be enough ?

**Reply by benoitgrelard** [reply]
Hey @Biki-das, yes a reproducible example in a sandbox would be great as at the moment I am unable to reproduce it.

**Comment by seanssel** [comment]
Any more info on this? I noticed the amount of shift seems dependent on the window size*. You can even see it happening on the radix docs pages (the effect is subtle here, watch this in fullscreen):


https://github.com/radix-ui/primitives/assets/2467750/f7b108dc-2914-4d45-aded-bec39b349671

**Reply by joaom00** [reply]
You can try https://github.com/radix-ui/primitives/issues/1925#issuecomment-1546419319

**Reply by seanssel** [reply]
> You can try [#1925 (comment)](https://github.com/radix-ui/primitives/issues/1925#issuecomment-1546419319)

No luck with any solution provided in that thread, plus there aren't any fixed position elements in my case. Strangely enough, despite seeing this on multiple browsers on both Mac/Windows on the radix dropdown doc page, in my application (electron - chromium 112.0.5615.49) I only see this on Windows.
